_includes/_help/data_faq/_posts/ads_curation_model.md

```diff
@@ -1,9 +1,9 @@
 The {{ include.site }} curation model is represented by the diagram below (see [Kurtz et al. 2021]({% if include.ads %}{{ site.ads_base_url }}{% else %}{{ site.scix_base_url }}{% endif %}/abs/2021BAAS...53d.470K/abstract)). 
```

Comment:
Whoops, I just noticed the `by the diagram below` in this sentence. I don't want to lose the link to the Kurtz paper - what if you did something like:
`The {{ include.site }} curation model is {% if include.ads %}represented by the diagram below (see [Kurtz et al. 2021]({{ site.ads_base_url }}/abs/2021BAAS...53d.470K/abstract)).{% else %} described in [Kurtz et al. 2021]({{ site.scix_base_url }}/abs/2021BAAS...53d.470K/abstract).{% endif %}`